lib/index.js

```diff
@@ -141,7 +157,10 @@ EslintValidationFilter.prototype.getDestFilePath = function getDestFilePath(rela
   }
 };
 
-EslintValidationFilter.prototype.processString = function processString(content, relativePath) {
+EslintValidationFilter.prototype.processString = function processString(
+  content,
+  relativePath
+) {
```

Comment:
can you remove the irrelevant formatting changes from this PR? that would make it much easier to review 🤔

Reply:
I can. Do you mind if I apply the formatting changes as a separate commit? (Just prettier's default since emacs is already hooked up)

Or do you wish to not see them entirely?

Reply:
I'd prefer it if they were a separate PR, with a separate discussion